pkgs/by-name/vm/vmlinux-to-elf/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  fetchFromGitHub,
+  lib,
+  python3,
+  unstableGitUpdater,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "vmlinux-to-elf";
+  version = "0-unstable-2024-07-20";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "marin-m";
+    repo = "vmlinux-to-elf";
+    rev = "da14e789596d493f305688e221e9e34ebf63cbb8";
+    hash = "sha256-GVoUIeJeLWCEFzrwiLX2h627ygQ7lX1qMp3hHT5O8O0=";
+  };
+
+  build-system = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+  ];
+
+  dependencies = [
+    python3.pkgs.lz4
+    python3.pkgs.python-lzo
+    python3.pkgs.zstandard
+  ];
+
+  pythonImportsCheck = [ "vmlinux_to_elf" ];
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = {
+    description = "Tool to recover a fully analyzable .ELF from a raw kernel, through extracting the kernel symbol table";
+    homepage = "https://github.com/marin-m/vmlinux-to-elf";
+    license = lib.licenses.gpl3Only;
+    mainProgram = "vmlinux-to-elf";
```

Comment:
New packages must have a maintainer.

Reply:
Please take a look at the other packages I'm trying to init to see if there are any other changes needed.
They will be all used as runtime dependencies for binwalk.

Reply:
I've had a look. Seems like Aleksanaa also started reviewing.